.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
+  "tabWidth": 2,
+  "singleQuote": false,
+  "trailingComma": "all",
+  "semi": true
+}
```

Comment:
Do we no longer use Deno tooling now?

Reply:
It used to help me to format files automatically. Should we also remove prettier from the recommended list of vscode extensions (and other rarely used extensions)?